build_tools/post_build_release_test.sh

```diff
@@ -12,7 +12,7 @@ REPO_ROOT="$(cd ${THIS_DIR?}/.. && pwd)"
 WHEELHOUSE_DIR="${REPO_ROOT?}/wheelhouse"
 
 # Set up environment.
-python -m venv "${WHEELHOUSE_DIR}"/test.venv
+python3.11 -m venv "${WHEELHOUSE_DIR}"/test.venv
```

Comment:
```suggestion
python3 -m venv "${WHEELHOUSE_DIR}"/test.venv
```

We might want to test not explicitly with 3.11 but in an environment where the `python` command is not available.

Reply:
I ran into issues with 3.10 (default on my Ubuntu), with some code found using Python 3.11+ features. Wanted to get the release scripting robust before tackling project-wide version support. Can look more closely...

Reply:
For me it would be okay to note down the minimum Python version in the docs but I am also fine if you prefer to pin it here for now. My intend was to avoid getting locked to exactly only one version.

Reply:
Switching back to just `python`.